test/optimizador_semanal.test.ts

```diff
@@ -0,0 +1,76 @@
+// Tests
+
+import { OptimizadorSemanal } from '../src/optimizador_semanal';
+import { Archivo } from '../src/archivo';
+
+describe('OptimizadorSemanal', () => {
+    it('debería crear un horario con actividades fijas', () => {
+        const optimizador = new OptimizadorSemanal();
+        const horario = new Archivo("./data/horario.txt");
+
+        optimizador.extraerHorario(horario);
+
+        optimizador.organizarHorario();
+
+        const expectedHorario = [
+            ["", "LUNES", "MARTES", "MIERCOLES", "JUEVES", "VIERNES"],
```

Comment:
Ah, ya veo por qué tiene 29 elementos. Digo lo mismo que antes, deberías usar tipos de datos específicos: o franja horaria, o diaria. En el objetivo 2 tenéis que hacer una modelización del problema y se trata precisamente de eso. Que tengas que imprimirlo así no implica que tengas que hacer un modelo incorrecto del problema.